backend/lib/deal-observer.js

```diff
@@ -15,10 +14,8 @@ import { Value } from '@sinclair/typebox/value'
  */
 export async function observeBuiltinActorEvents (blockHeight, pgPool, makeRpcRequest) {
   const eventType = 'claim'
-  const activeDeals = await getActorEvents(getActorEventsFilter(blockHeight, eventType), makeRpcRequest)
-  assert(activeDeals !== undefined, `No ${eventType} events found in block ${blockHeight}`)
-  console.log(`Observed ${activeDeals.length} ${eventType} events in block ${blockHeight}`)
```

Comment:
Can we please preserve this line? I found it useful to see in the logs how many deals we are adding per block and which loop iterations don't find any new blocks.

Again, this is not a blocker.